beast/observationmodel/ast/make_ast_xy_list.py

```diff
@@ -61,6 +62,13 @@ def pick_positions_from_map(
         put in a bin, so that a set of tiles of the map is obtained for
         each range of source density/background values.
 
+    bin_width: int
```

Comment:
Also, it might be helpful if it is possible to pick the scale for the bin_width: linear vs. log. We still need to investigate which binning scale would be better.